File: app/src/main/java/com/ahl/annahockeyleague/Entity/TopScorers.java
package com.ahl.annahockeyleague.Entity;

import org.bson.types.ObjectId;

public class TopScorers {

    private ObjectId playerId;
    private int goals;
    private Player player;
    private Team team;

    public ObjectId getPlayerId() {
        return playerId;
    }

    public void setPlayerId(ObjectId playerId) {
        this.playerId = playerId;
    }

    public Player getPlayer() {
        return player;
    }

    public void setPlayer(Player player) {
        this.player = player;
    }

    public int getGoals() {
        return goals;
    }

    public void setGoals(int goals) {
        this.goals = goals;
    }

    public Team getTeam() {
        return team;
    }

    public void setTeam(Team team) {
        this.team = team;
    }
}

File: app/src/main/java/com/ahl/annahockeyleague/kotlin/data/TopScorers.kt
package com.ahl.annahockeyleague.kotlin.data

import com.ahl.annahockeyleague.Entity.Player
import com.ahl.annahockeyleague.Entity.Team
import org.bson.types.ObjectId

class TopScorers(val playerId: ObjectId,
                val goals: Int,
                val player: Player,
                val team: Team)